pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
In Kubernetes, we have an "insecure port".  I was thinking that the initial policy would be created via the insecure port.  The insecure port currently does not do any authentication or authorization.  However, we could change the handler path for the insecure port so that it sets the username to a reserved field, like: `system:apiserver-insecure-port-user`, and special case that user in the validation logic?